File: OCH_Tao/ABC/B/232.py
S = list(map(lambda x:ord(x)-97,list(input())))
T = list(map(lambda x:ord(x)-97,list(input())))
diff = S[0]-T[0]
for i in range(len(S)):
  if (S[i]-diff)%26!=T[i]:
    print("No")
    break
else:
  print("Yes")

File: OCH_Tao/ABC/B/232.js
const [S, T] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n").map(x => [...x].map(y => y.charCodeAt(0) - 97));
const diff = S[0] - T[0];
for (let i = 0; i < S.length; i++) {
  if ((26 + S[i] - diff) % 26 !== T[i]) {
    console.log("No");
    process.exit();
  }
}
console.log("Yes");